docs/console/docs/06-01-microfrontend.md

```diff
@@ -3,7 +3,7 @@ title: MicroFrontend
 type: Custom Resource
 ---
 
-The `microfrontends.ui.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to extend the Kyma Console. It allows you to extend the Console for the specific Namespace. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+The `microfrontends.ui.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to extend the Kyma Console. It allows you to extend the Console for the specific Namespace. MicroFrontends get added to console automatically based on the yaml. In order to avoid naming colissions with the core system, the root node gets prefixed with `mf-` in the URL. Additionally `navigationContext` and `viewGroup` [Node configuration](https://github.com/kyma-project/luigi/blob/master/docs/navigation-parameters-reference.md#node-parameters) options are set, to allow simple navigation. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
```

Comment:
```suggestion
The `microfrontends.ui.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to extend the Kyma Console. It allows you to extend the Console for the specific Namespace. The MicroFrontend CR is added to the Console automatically based on the `yaml` file. To avoid naming conflicts with the core system, the root node receives the `mf-` prefix in the URL. Additionally, the **navigationContext** and **viewGroup** [node configuration](https://github.com/kyma-project/luigi/blob/master/docs/navigation-parameters-reference.md#node-parameters) parameters are set to allow simple navigation. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
```

Reply:
I believe In this context we should use sth like:
```
A micro frontends is added to console ....
```

Reply:
Yes. you are right. Changing the suggestion.